### PR TITLE
Bases for the first four Hub01 basement npcs

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Content "Add the first four Hub 01 basement Npcs"

#### Purpose of change
Add the bases for the first four NPCs, This is mostly just their basic classes and gear, alongside with the barebones first meet dialogue.

To later be expanded with their complete respective dialogue.

#### Testing

Game loads the Npcs correctly

### CODE REVIEW

**Comment by Maleclypse** [comment]
Missing an ellipsis at the end of the summary.

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -6486,6 +6486,18 @@
...
+    "valid": false,
+    "purifiable": false,
+    "profession": true
+  },
+  {
+    "type": "mutation",
+    "id": "PROF_HUB01_ANCILLIARY",
+    "name": { "str": "Hub 01 Ancilliary" },
+    "points": 0,
+    "description": "You are known to be a capable and resourcefull mercenary at the employ of Hub 01.",
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:6499]:
```suggestion
    "description": "You are known to be a capable and resourceful mercenary at the employ of Hub 01.",
```

File: data/json/npcs/robofac/robofac_basement_npcs/NPC_Darla_Novak.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    "common": false,
+    "//": "HVAC maintenance.  Keeps melchior from boiling to death and you from choking on stale bunker air.",
+    "bonus_str": { "rng": [ -1, 2 ] },
+    "bonus_dex": { "rng": [ -1, 2 ] },
+    "bonus_int": { "rng": [ -2, 2 ] },
+    "bonus_per": { "rng": [ -1, 2 ] },
+    "worn_override": "HUB01_DNovak_worn",
+    "carry_override": "HUB01_DNovak_carried",
+    "weapon_override": "HUB01_DNovak_wield",
+    "traits": [ { "group": "Appearance_demographics" } ],
```
Comment (Maleclypse) [inline data/json/npcs/robofac/robofac_basement_npcs/NPC_Darla_Novak.json:28]: Does Appearance Demographics check gender above?

File: data/json/npcs/robofac/robofac_basement_npcs/NPC_Jonathan_Farrier.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  {
+    "type": "item_group",
+    "id": "HUB01_JFarrier_wield"
+  },
+  {
+    "type": "talk_topic",
+    "id": "HUB01_JFarrier_Intro",
+    "dynamic_line": {
+      "u_has_trait": "PROF_HUB01_ANCILLIARY",
+      "yes": "Aw, hey, you must be that new blood runnin' jobs up top.  So, they finally let you down, eh?  My names Jon, Jon Farrier.  Nice to meet you, real nice.",
```
Comment (Night-Pryanik) [inline data/json/npcs/robofac/robofac_basement_npcs/NPC_Jonathan_Farrier.json:65]:
```suggestion
      "yes": "Aw, hey, you must be that new blood runnin' jobs up top.  So, they finally let you down, eh?  My name's Jon, Jon Farrier.  Nice to meet you, real nice.",
```

File: data/json/npcs/robofac/robofac_basement_npcs/NPC_Darla_Novak.json (hunk 2 above)
Comment (John-Candlebury) [reply data/json/npcs/robofac/robofac_basement_npcs/NPC_Darla_Novak.json:28]: Hmm good question, let me check

Comment (John-Candlebury) [reply data/json/npcs/robofac/robofac_basement_npcs/NPC_Darla_Novak.json:28]: So it doesnt include beards or anything so I guess it does.

**Pull request merged**